### ISSUE TITLE
Unable to delete components in ocp extension

### ISSUE THREAD

**Issue by bkemp** [redhat-developer/vscode-openshift-tools#2543]
Able to delete the 'project' but not able to delete and remove a component in the extension sidebar.

**Comment by bkemp** [comment]
additionally, after deleting the namespace/project, deploying the orphaned component to ocp silently does nothing

**Comment by dgolovin** [comment]
> Able to delete the 'project' but not able to delete and remove a component in the extension sidebar.

@bkemp You are saying you cannot deleted component in  'Components View'. Is that correct? 

>   deploying the orphaned component to ocp silently does nothing

There is no deployment feature in extension, could you elaborate what exactly you are using in UI?

Thank you.

**Comment by bkemp** [comment]
Yes, the component cannot be deleted AND there is an action in the ellipsis '...' that says 'Deploy to Openshift'. This is after the cluster does not exist, as well. I can click on it, but nothing happens, of course. It's very misleading.

**Comment by datho7561** [comment]
"Deploy to OpenShift" was a vscode-quarkus feature. We've removed that feature from vscode-quarkus, since it wasn't working. This what the "Delete Component" options look like now:
![Screenshot from 2023-10-19 10-25-25](https://github.com/redhat-developer/vscode-openshift-tools/assets/22376627/625d7751-2aee-4291-a5dd-96e344da3449)
